src/dfcx_scrapi/tools/evaluations.py

```diff
@@ -94,18 +94,21 @@ def __init__(
             agent_id=self.agent_id, tools_map=tools_map, creds=self.creds)
         self.ar = AgentResponse()
         self.ch = ConversationHistory(agent_id=self.agent_id, creds=self.creds)
-
+        self.project_id = self.get_project_id_from_agent_id("agent", agent_id)
```

Comment:
If we make the changes in `get_project_id_from_agent_id` then we can remove the "agent" arg here and just pass `agent_id`